src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -143,12 +177,37 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                         {
                             id: "marginInVotes",
                             Header: "Margin i stemmer",
-                            accessor: (d: PartyResult) => vulnerableMap.get(d.partyCode),
+                            accessor: (d: PartyResult) => vulnerableMap.get(d.partyCode)!,
+                            Cell: (row) => {
+                                if (row.original.partyCode === vulnerableVotes.partyCode) {
+                                    const cellColor =
+                                        row.original.partyCode === vulnerable.runnerUp.partyCode ? "#ffdd57" : "orange";
```

Comment:
Why "match" these colours? And why with custom CSS rather than the actual classes being used?